### ISSUE TITLE
Fix the Vote Execution Delay Forum Post so it works for Disputable voting

### ISSUE THREAD

**Issue by GriffGreen** [TECommons/coordination#472]

**Comment by divine-comedian** [comment]
altered text a bit in[Vote Execution Delay]( https://forum.tecommons.org/t/vote-execution-delay-and-what-it-means/130) to distinguish dandelion voting - need some more clarity of disputable voting vote execution delay @GriffGreen . 

Disputable Voting there is no rage quit because of the ABC right? So people can exit any time they want following what the 'sell curve' is and the decided exit tribute is?

**Comment by freedumbs00** [comment]
You're right.  There is no "Rage Quit" once the curve is launched.   The dandelion is the only instance it can be acted.

**Comment by divine-comedian** [comment]
https://docs.google.com/document/d/1__kysKxgqtArILNc-1bJ6i8Oxq_ogrq7NEg1ZGS8hx8/edit?usp=sharing
